etc/org.opencastproject.distribution.streaming.wowza.WowzaStreamingDistributionService.cfg

```
@@ -1,7 +1,7 @@
 # Configures the streaming directory path for Wowza streaming service.
 # If this configuration is not set default path will be set
 # Default: streams folder in Opencast storage folder(org.opencastproject.storage.dir in custom.properties file)
-org.opencastproject.streaming.directory=~/opencast/streams
+#org.opencastproject.streaming.directory=/srv/opencast/streams
```

Maybe this will work:

```suggestion
org.opencastproject.streaming.directory=${org.opencastproject.storage.dir}/streams
```